pkg/runner/run_context.go

```diff
@@ -102,6 +102,17 @@ func (rc *RunContext) GetBindsAndMounts() ([]string, map[string]string) {
 		name + "-env":   ActPath,
 	}
 
+	for _, v := range rc.Run.Job().Container().Volumes {
```

Comment:
`Container()` can be `nil` and panic here, same for `Job()`

Reply:
I think `Container()` can be `nil` if workflow doesn't use a container, but how `Job()` can be `nil`?
Are two if statements with `...() != nil` sufficient?

Reply:
fixed on aec56fc5306f1374e9a87aa5f2b4db34be9eac41